cds/modules/deposit/mappings/deposits/records/videos/video/video-v1.0.0.json

```diff
@@ -214,6 +214,17 @@
             }
           }
         },
+        "related_links": {
+          "type": "object",
```

Comment:
It is an array, that's why it's marked as `object` :) (God, I love the simplicity and transparency of using JSON for our forms)